CHANGES.md

```diff
@@ -25,6 +25,8 @@ unreleased
 - Update description to reflect that `ppxlib` contains more than a library
   (#422, @pitag-ha)
 
+- Add support for OCaml 5.1 (#428, @shym, @Octachron , @pitag-ha, @panglesd)
```

Comment:
```suggestion
- Add support for OCaml 5.1, excluding OCaml `5.1.0~alpha1` (#428, @shym, @Octachron , @pitag-ha, @panglesd)
```

Since it isn't common to be incompatible with alpha-releases, I think it's worth pointing that out.